File: app/src/main/java/com/codingblocks/codingblocks/models/BookData.java

package com.codingblocks.codingblocks.models;

import java.util.List;
import javax.annotation.Generated;
import com.google.gson.annotations.SerializedName;

@Generated("net.hexar.json2pojo")
@SuppressWarnings("unused")
public class BookData {

    @SerializedName("progress")
    private Progress mProgress;
    @SerializedName("sections")
    private List<Section> mSections;
    @SerializedName("version")
    private String mVersion;

    public Progress getProgress() {
        return mProgress;
    }

    public void setProgress(Progress progress) {
        mProgress = progress;
    }

    public List<Section> getSections() {
        return mSections;
    }

    public void setSections(List<Section> sections) {
        mSections = sections;
    }

    public String getVersion() {
        return mVersion;
    }

    public void setVersion(String version) {
        mVersion = version;
    }

}

File: app/src/main/java/com/codingblocks/codingblocks/models/BookData.kt
package com.codingblocks.codingblocks.models

import javax.annotation.Generated
import com.google.gson.annotations.SerializedName
import kotlin.collections.List

@Generated("net.hexar.json2pojo")
data class BookData(

        @SerializedName("progress")
        var progress: Progress,
        @SerializedName("sections")
        var sections: List<Section>,
        @SerializedName("version")
        var version: String

)
